### ISSUE TITLE
Atom tabs switch-list does not appear

### ISSUE THREAD

**Issue by jonyeezs** [atom-material/atom-material-ui#359]
Using Atom's tab package for tab-switching, had the tab switch disabled.

The class on that element is `select-list tabs-mru-switcher`

**Reproduction Steps:**

**Settings for atom-material-ui:**

- [x] Generate complementary accent
- [ ] Paint text editor's cursor
- [ ] Use animations
- [x] Contrasting panels
- [ ] Panels cast shadows
- [x] Compact tab bar
- [x] No minimum width for tabs
- [ ] Tinted tab bar
- [ ] Blend tree view with tab bar
- [x] Compact tree view

**Settings for tabs:**
- [ ] Add New Tabs At End
- [x] Always Show Tab Bar
- [x] Enable MRU Tab Switching
- [x] Enable VSC Coloring
- [x] Show Icons
- Tab Scrolling - platform
- [x] Use Preview Tabs

Using ctrl-tab to switch tabs
**keybindings**
```
'body':
  'ctrl-tab': 'pane:show-next-recently-used-item'

'body':
  'ctrl-tab ^ctrl': 'pane:move-active-item-to-top-of-stack'
```

**Expected behavior:**
![](https://cloud.githubusercontent.com/assets/553742/20579681/075e7d00-b183-11e6-80c9-d61b7986e9fe.gif)

**Observed behavior:**

![](http://i.giphy.com/xTiN0m5vEU4KfoNM2I.gif)

**Atom version:** v1.14.3
**OS and version:** Windows 10

**Installed packages:**

https://atom.io/packages/tabs

**Comment by tumido** [comment]
Let's wait what @silvestreh will show us when the sneaky 2.0 branch is merged and released. ;) I was wondering, when I was reading the 1.14 Atom release notes, why this list is not showing up for me, so I found your ticket and investigated the issue. This is a workaround, I've been able to come up with:

1. Open the `lib/tabs-settings.js` file (located in this package sources in `.atom/packages/atom-material-ui/` folder)
2. Remove lines 41 to 47 containing:
```
atom.workspace.onDidChangeActivePaneItem(() => {
    var activeTab = document.querySelector('.tab-bar .tab.active');

    if (activeTab && activeTab.click) {
        activeTab.click();
    }
});
```
3. Reload your Atom window via `window:reload` (or restart Atom)

The issue is, the code mentioned above is clicking on the newly selected tab (see `activeTab.click();`). That effectively hides the tabs list, because before it can be rendered, a new tab is "fully activated" and the list is not necessary anymore.

I've tried to figure out if removing the code has any impact on the package, but it behaves just fine for me and no other problems are introduced.

In addition, I would like to say that this should not be used as a fix as is. I pretty much enjoy the behavior when tab list is not shown. It feels a bit quicker to me. So I would be glad, if this can be introduced just as an option, via some toggle in settings. I actually like this bug, lol.

**Comment by silvestreh** [comment]
It's on the `2.0` branch, but requires Atom 1.17 (unfortunately, not even in the Beta channel)

<img width="1440" alt="captura de pantalla 2017-04-07 a las 22 45 56" src="https://cloud.githubusercontent.com/assets/1133339/24824570/0136c29a-1be4-11e7-9f1e-a64db86c17d5.png">

I'll leave this issue open!

**Comment by silvestreh** [comment]
This is fixed in `2.0` which is available for Atom 1.17-beta users.
